### ISSUE TITLE
Does this work with jQuery? 

### ISSUE THREAD

**Issue by feniixx** [VincentGarreau/particles.js#85]
I followed all the steps but it's not displaying the particles neither the lines, there's no error output...

**Comment by VincentGarreau** [comment]
Yes it works with jQuery. Please show a demo or something to resolve your bug.
